InSimDotNet/Packets/LocalCarLights.cs

```diff
@@ -0,0 +1,113 @@
+﻿namespace InSimDotNet.Packets
+{
+    /// <summary>
+    /// Switches for CAR LIGHTS <see cref="IS_SMALL"/> SMALL_LCL
+    /// </summary>
+    public static class LocalCarLights
+    {
+        /// <summary>
+        /// Bit 0
+        /// </summary>
+        private const int LCL_SET_SIGNALS = 1;  // bit 0
+
+        /// <summary>
+        /// Bit 1
+        /// </summary>
+        private const int LCL_SPARE_2 = 1 << 1;  // bit 1
+
+        /// <summary>
+        ///  bit 2
+        /// </summary>
+        private const int LCL_SET_LIGHTS = 1 << 2;      // bit 2
+
+        /// <summary>
+        /// Bit 3
+        /// </summary>
+        private const int LCL_SPARE_8 = 1 << 3;  // bit 3
+
+        /// <summary>
+        /// Bit 4
+        /// </summary>
+        private const int LCL_SET_FOG_REAR = 1 << 4; // bit 4
+
+        /// <summary>
+        /// Bit 5
+        /// </summary>
+        private const int LCL_SET_FOG_FRONT = 1 << 5;    // bit 5
+
+        /// <summary>
+        /// Bit 6
+        /// </summary>
+        private const int LCL_SET_EXTRA = 1 << 6;	// bit 6
+
+        /// <summary>
+        /// Turn signals off
+        /// </summary>
+        public const int LCL_SIGNALS_OFF = LCL_SET_SIGNALS;
+
+        /// <summary>
+        /// Turn signal left
+        /// </summary>
+        public const int LCL_SIGNALS_LEFT = (1 << 16) + LCL_SET_SIGNALS;
```

Comment:
suggestion: use `|` for bitwise flag operations (for all applicable flags)

Reply:
I just followed the existing style, but I agree is better. Changed.